src/game/AI/ScriptDevAI/scripts/kalimdor/wailing_caverns/wailing_cavernsScripts.cpp

```diff
@@ -60,27 +59,76 @@ enum
     SPELL_SHAPESHIFT        = 8153,
 
     NPC_DEVIATE_RAPTOR      = 3636,                         // 2 of them at the first stop
-    NPC_DEVIATE_VIPER       = 5755,                         // 3 of them at the circle
+    NPC_DEVIATE_VIPER       = 5755,                         // 3 RND at the circle
+    NPC_DEVIATE_ADDER       = 5048,                         // 3 RND at the circle
     NPC_DEVIATE_MOCCASIN    = 5762,                         // 6 of them at Naralex chamber
     NPC_NIGHTMARE_ECTOPLASM = 5763,                         // 10 of them at Naralex chamber
-    NPC_MUTANUS             = 3654
+    NPC_MUTANUS             = 3654,
+
+    PATH_ID_NARALEX         = 4300000,                      // Sniffed Waypoints for Escort Event
```

Comment:
This has to be changed to 3678, else we're good.
ScriptDev waypoint_path will always use creature entry as pathid.